fog/ledger/server/src/key_image_store_server.rs

```diff
@@ -200,6 +204,11 @@ where
             .expect("failed starting report cache thread"),
         );
 
+        self.metrics_update_thread = Some(
+            MetricsUpdateThread::start(self.ledger.clone(), self.logger.clone())
+                .expect("failed starting metrics update thread"),
+        );
+
```

Comment:
Added a metrics updater to the stores for symmetry.
One can get some of these values from https://github.com/mobilecoinfoundation/mobilecoin/blob/09062b26226eb6df3cdee082573c4faca2388bb6/fog/ledger/server/src/db_fetcher.rs#L285 but it's a different counter/gauge than the other ones.

Reply:
re-worked no longer using a dedicated thread for this.